.vscode/launch.json

```diff
@@ -600,6 +611,21 @@
             "type": "coreclr",
             "request": "attach",
             "processId": "${command:pickProcess}"
+        },
+        {
+            "name": "run-MsSqlMigratorUtility",
+            "presentation": {
+                "hidden": true,
+            },
+            "requireExactSource": true,
+            "type": "coreclr",
+            "request": "launch",
+            "program": "${workspaceFolder}/util/MsSqlMigratorUtility/bin/Debug/net8.0/MsSqlMigratorUtility.dll",
+            "args": [
+                "Server=localhost;Database=vault_dev;User Id=SA;Password=SET_A_PASSWORD_HERE_123;Encrypt=True;TrustServerCertificate=True;"
```

Comment:
⚠️  This password would need to change though, and couldn't be maintained here ... the utility has to be able to read the user secrets in `dev/`.

Reply:
Yes, but how could the user of this launcher use this without modifying the `args`? Maybe I am missing something but we can't have to ask users to modify this file to get this to run.